### PR TITLE
Add Pub-Sub for wallet and trasaction history

### PR DESCRIPTION
**Pull request by Weber-H2O2**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -42,6 +42,7 @@
     "standard-ecies": "^2.0.0",
     "totp.js": "^0.0.1",
     "typings": "^2.1.1",
-    "vue-axios": "^3.3.7"
```
Comment (eigmax) [inline package.json:45]: What's the difference between  vue-axios and axios?

File: src/wallet/wallet.ts [hunk 2]
```diff
@@ -1,10 +1,213 @@
...
 
-import { Op } from "sequelize";
+// Records txid => wallet, wallet is a Sequelize model which can be used to update status
+let TRANSACTION_ADD_SIGNER_BY_OWNER_MAP = new Map();
+let TRANSACTION_ADD_SIGNER_BY_SIGNER_MAP = new Map();
+let TRANSACTION_DELETE_SIGNER_MAP = new Map();
+
+// TODO: Pubsub.subscribeOnce is better? Maybe we should think about it
+
+function addWalletStatusSubscriber(txid, wallet_id) {
```
Comment (eigmax) [inline src/wallet/wallet.ts:23]: We can use async to polish the code.

File: package.json (hunk 1 above)
Comment (Weber-H2O2) [reply package.json:45]: The package is not needed here. Should be removed.

File: src/wallet/wallet.ts (hunk 2 above)
Comment (Weber-H2O2) [reply src/wallet/wallet.ts:23]: Seems that `PubSub` only accepts `function` not `async function`. `PubSub.subscribe` is async call actually.

**Review by eigmax — APPROVED** [review]

**Pull request merged**